packages/react-app/src/components/pages/Bounties/Form/index.tsx

```diff
@@ -101,9 +102,25 @@ const Form = ({ bountyForm }: { bountyForm: any }): JSX.Element => {
       console.error(error)
     }
   }
+
+  const publishBountyToDiscordChannel = (values: any): void => {
+    const embedMessage = DiscordUtils.generateBountyEmbedsMessage(values)
+    fetch(process.env.NEXT_PUBLIC_DISCORD_BOUNTY_BOARD_WEBHOOK as string, {
```

Comment:
Is the webhook meant to be public? Would spammers abuse it, or am I not understanding it correctly?

A safer option might be to move it out of public and let the API handle it in [`src/pages/api/bounties/[id].ts`](https://github.com/BanklessDAO/bounty-board/blob/1ef86e998dd9136cde412fdf3cb11015762341c2/packages/react-app/src/pages/api/bounties/%5Bid%5D.ts#L39). This would keep anyone from sniffing it out of the client side JS code.

Reply:
done!